lib/polaris/view_components/version.rb

```diff
@@ -1,5 +1,5 @@
 module Polaris
   module ViewComponents
-    VERSION = "2.1.0"
```

Comment:
Version is updated automatically during gem release. We don't need to change it manually.

Reply:
Oh, yes. It was my mistake 🤦

Reply:
This is reverted